engine/src/flutter/lib/gpu/shader.h

```diff
@@ -81,6 +83,7 @@ class Shader : public RefCountedDartWrappable<Shader> {
   std::unordered_map<std::string, UniformBinding> uniform_structs_;
   std::unordered_map<std::string, TextureBinding> uniform_textures_;
   std::vector<impeller::DescriptorSetLayout> descriptor_set_layouts_;
+  bool is_first_time_;
```

Comment:
Initialize scalars in the header in case the ctor misses them (or we add a new ctor).